src/components/app/auth/login.tsx

```diff
@@ -0,0 +1,22 @@
+import Link from 'next/link';
+
+import { Button } from '@/components/ui/button';
+import { Input } from '@/components/ui/input';
+
+export const Login = () => {
+  return (
+    <main className="flex h-screen items-center justify-center">
+      <div className="flex w-[300px] flex-col space-y-3">
+        <h3>Login</h3>
+        <p>Login to Indonesia Open Course</p>
+        <Input type="email" placeholder="Enter your email" />
+        <Input type="password" placeholder="Enter your password" />
```

Comment:
Hi, shouldn't we put `<input>` inside `<form>` attribute?

Reply:
I don't usually use the `<form>` tag.

I usually code it like this

```tsx
  const [loginData, setLoginData] = useState({
    email: "",
    password: "",
  });
```

then a  handleEventChange function
```tsx
  const handleEventChange = (event) => {
    const { name, value } = event.target;
    setLoginData({ ...loginData, [name]: value });
  };
```

then a handleSubmit function
```tsx
  const { email, password } = loginData;
  const res = await fetch(`${API_URL}/login`, {
    method: "POST",
    body: JSON.stringify({ email, password }),
    });
  const { message, token, role, error } = await res.json();
```

Reply:
But should we put `<form>` tag in this case? @ahmadrasyidsalim